lib/decidim_staging/budgets_groups_workflow_pam2020.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module DecidimStaging
+  # Specific Workflow for Barcelona's 2020 PAM
+  class BudgetsGroupsWorkflowPam2020 < Decidim::Budgets::Groups::Workflows::Base
+    PAM2020AUTHORIZATIONHANDLER = 'dummy_census_authorization_handler'
+
+    # The budget component in the user's scope is highlighted.
+    def highlighted?(component)
+      component == user_scope_component unless voted?(user_scope_component)
```

Comment:
I'd change this with `... unless user_scope_component && voted?(user_scope_component)`